multi_map_it_test.go

```diff
@@ -539,6 +539,25 @@ func TestMultiMap_ContainsEntry(t *testing.T) {
 	})
 }
 
+func TestMultiMap_ValueCount(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		key := "key"
+		targetValues := []interface{}{"v1", "v2", "v3"}
+		it.Must(m.PutAll(ctx, key, targetValues...))
+		count, err := m.ValueCount(ctx, key)
+		if err != nil {
+			t.Error(err)
+		}
+		assert.EqualValues(t, count, len(targetValues))
+		nonExistingKey := "dummyKey"
+		count, err = m.ValueCount(ctx, nonExistingKey)
+		if err != nil {
+			t.Error(err)
+		}
+		assert.EqualValues(t, count, 0)
```

Comment:
One small suggestion, as I see most of the assertion functions follow `Assert.Equal(exptected, actual)` order. It seems same in go. https://pkg.go.dev/github.com/stretchr/testify/assert#EqualValues

Don't you think would be better?

Reply:
Good point.
**Short answer:** I use `EqualValue` because I don't interested in type of the `count`. 

**Let me clarify in a scenario,**
Suppose that the return value of the `ValueCount` is changed from `int` to `uint32` in the API. If I used `Equal` it would just fail because it checks both type and value like `reflect.DeepEqual()`. That creates an extra work because I need to change the test itself to pass.

Good discussion: https://github.com/stretchr/testify/issues/235

Reply:
@utku-caglayan thanx for clarification. Yes, I meant that.

Reply:
Sorry for misunderstanding. You are both right. I'm changing.